File: OOP_Variable/1.py
class C(object):
    def __init__(self, v1):
        self.value = v1

    def show(self):
        print(self.value)


#파이썬: 인스턴스변수에 접근이 허용됨
c1 = C(10)
print(c1.value)
c1.value = 20
print(c1.value)
c1.show()

File: OOP_Variable/1.rb
class C
  def initialize(v1)
    @value = v1
  end
  def show()
    p @value
  end
end
#루비: 인스턴스변수에 접근이 허용되지 않음
c1 = C.new(10)
#p c1.value     # 허용X
# c1.value = 20 # 허용X
c1.show()
